wit/wit.py

```diff
@@ -71,10 +71,12 @@ def __init__(self, access_token, actions=None, logger=None):
             self.logger.error('Stories and POST /converse have been deprecated. This will break in February 2018!')
             self.actions = validate_actions(self.logger, actions)
 
-    def message(self, msg, context=None, verbose=None):
+    def message(self, msg, context=None, num=None, verbose=None):
```

Comment:
`num -> n`, let's keep the naming consistent between the Python keyword arguments and the HTTP API